ios/TestApp/README.md

```diff
@@ -44,6 +44,85 @@ python test/mobile/model_test/gen_test_model.py ios-test
 cd ios/TestApp/benchmark; python coreml_backend.py
 ```
 
+## Run test on AWS Device Farm
+The test app and its test suite could also be run on actual devices via
+AWS Device Farm.
+
+1. Checkout PyTorch repo including all submodules
+
+2. Build PyTorch for iOS devices, not for simulator
```

Comment:
Do you need to do this on MacOS? Do you need to have XCode installed?

Reply:
Oh yeah, you need to run this on MacOS with XCode installed.  Let me clarify this as a prerequisite